aten/src/ATen/cpu/vec256/vec256_qint.h

```diff
@@ -128,6 +128,48 @@ struct Vec256<c10::qint8> {
         return retval;
     }
 
+    Vec256<c10::qint8> relu(Vec256<c10::qint8> zero_point) {
+#ifdef __AVX2__
+      return _mm256_max_epi8(vals, zero_point.vals);
+#else
+      // Pray the compiler can autovectorize this
```

Comment:
There is way to vectorize it using _mm_max_epi8 (SSE 4.1) and we can avoid load/stores. I think it's ok leave it as is for pre-avx2 machines. Just curious: Do we support pre-SSE4.1 machines with PyTorch?

Reply:
Yeah I wrote a routine using that but it got to be very long and unwieldy and I thought it wasn’t worth it to continue

Reply:
I don't think it's well-specified which machines we officially support in PyTorch. Defining that would likely make writing code like this a lot cleaner

@soumith any thoughts here?

Reply:
the default binary build is compiled for atleast the user having SSE4